Microsoft.Toolkit.Uwp.SampleApp/Models/Sample.cs

```diff
@@ -147,22 +147,26 @@ public bool IsSupported
             }
         }
 
-        public async Task<string> GetDocumentationAsync()
+#pragma warning disable SA1009 // Doesn't like ValueTuples.
+        public async Task<(string contents, string Path)> GetDocumentationAsync()
```

Comment:
Both return variables should be cased the same. Either capitalize both, or lowercase both